### PR TITLE
doc: update CHANGELOG for v1.33.0 release

### PR DESCRIPTION
**Pull request by scotthart**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -97,20 +99,87 @@
...
+spell this inline namespace in their own code. This has been corrected. Our
+Doxygen documentation no longer shows the versioned inline namespace name;
+instead, it shows users how to correctly spell our symbol names without
+referencing the versioned inline namespace.
 
 [storage-dox-link]: https://googleapis.dev/cpp/google-cloud-storage/latest/
 
+### [BigQuery](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigquery/README.md)
+
+**BREAKING CHANGE**
```
Comment (coryan) [inline CHANGELOG.md:111]: This is not breaking, we defined `gcpcxxV1` as a backwards compatibility alias.  The break will be in 1 year from now.

File: CHANGELOG.md [hunk 2]
```diff
@@ -97,20 +99,87 @@
...
+* fix(storage): respect ContentType in Client::UploadFile ([#7521](https://github.com/googleapis/google-cloud-cpp/pull/7521))
+* doc: remove inline namespace from doxygen ([#7461](https://github.com/googleapis/google-cloud-cpp/pull/7461))
+* fix(storage): prevent crashes on double Close() ([#7390](https://github.com/googleapis/google-cloud-cpp/pull/7390))
+* feat(storage): add bucket attributes for RPO ([#7384](https://github.com/googleapis/google-cloud-cpp/pull/7384))
+* doc(storage): 'unspecified' value for PAP is deprecated ([#7377](https://github.com/googleapis/google-cloud-cpp/pull/7377))
+
+### [Common Libraries](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/README.md)
+
+* fix(common): fewer crashes with dynamic loading ([#7512](https://github.com/googleapis/google-cloud-cpp/pull/7512))
+* doc(storage): label `ClientOptions` as deprecated ([#7511](https://github.com/googleapis/google-cloud-cpp/pull/7511))
```
Comment (coryan) [inline CHANGELOG.md:176]: I would think this is a change for the storage library?

File: CHANGELOG.md [hunk 3]
```diff
@@ -97,20 +99,87 @@
...
+* feat(common): GUAC for async read-write streams ([#7442](https://github.com/googleapis/google-cloud-cpp/pull/7442))
+* feat(pubsub): implement GUAC for Publisher ([#7440](https://github.com/googleapis/google-cloud-cpp/pull/7440))
+* feat(pubsub): implement GUAC for SchemaAdmin ([#7436](https://github.com/googleapis/google-cloud-cpp/pull/7436))
+* feat(pubsub): implement GUAC for SubscriptionAdmin ([#7432](https://github.com/googleapis/google-cloud-cpp/pull/7432))
+* feat(pubsub): implement GUAC for TopicAdmin ([#7428](https://github.com/googleapis/google-cloud-cpp/pull/7428))
+* fix(pubsub): do not overwrite ConnectionOptions ([#7406](https://github.com/googleapis/google-cloud-cpp/pull/7406))
+
+### [Spanner](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/spanner/README.md)
+
+**BREAKING CHANGE**
```
Comment (coryan) [inline CHANGELOG.md:156]: Ditto, I think we should not mark these changes as breaking because they are not (yet) breaking.

**Review by coryan — CHANGES_REQUESTED** [review]

**Review by scotthart — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @coryan)

___
*[CHANGELOG.md, line 111 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546#-MnRV6JcvMPNH-gJz2qe-r1-111:-MnR_NcqBWv4eR2VHQXX:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/8475d5384148b55bd29209b5112ea2e23ca77c65/CHANGELOG.md#L111)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

This is not breaking, we defined `gcpcxxV1` as a backwards compatibility alias.  The break will be in 1 year from now.
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 156 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546#-MnRV6JcvMPNH-gJz2qe-r1-156:-MnR_OC95ntbXQkjvGEu:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/8475d5384148b55bd29209b5112ea2e23ca77c65/CHANGELOG.md#L156)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

Ditto, I think we should not mark these changes as breaking because they are not (yet) breaking.
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 176 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546#-MnRV6JcvMPNH-gJz2qe-r1-176:-MnR_OWt3uzEWOxA4Os6:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/8475d5384148b55bd29209b5112ea2e23ca77c65/CHANGELOG.md#L176)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

I would think this is a change for the storage library?
</blockquote></details>

Done.

File: CHANGELOG.md [hunk 4]
```diff
@@ -97,20 +99,72 @@
...
+### [Pub/Sub](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/pubsub/README.md)
+
+* feat: create relocatable pkg-config files ([#7481](https://github.com/googleapis/google-cloud-cpp/pull/7481))
+* doc: remove inline namespace from doxygen ([#7461](https://github.com/googleapis/google-cloud-cpp/pull/7461))
+* feat(pubsub): implement GUAC for Subscriber ([#7449](https://github.com/googleapis/google-cloud-cpp/pull/7449))
+* feat(common): GUAC for async read-write streams ([#7442](https://github.com/googleapis/google-cloud-cpp/pull/7442))
+* feat(pubsub): implement GUAC for Publisher ([#7440](https://github.com/googleapis/google-cloud-cpp/pull/7440))
+* feat(pubsub): implement GUAC for SchemaAdmin ([#7436](https://github.com/googleapis/google-cloud-cpp/pull/7436))
+* feat(pubsub): implement GUAC for SubscriptionAdmin ([#7432](https://github.com/googleapis/google-cloud-cpp/pull/7432))
+* feat(pubsub): implement GUAC for TopicAdmin ([#7428](https://github.com/googleapis/google-cloud-cpp/pull/7428))
```
Comment (dbolduc) [inline CHANGELOG.md:141]: These can probably be combined, and I think the common one can be dropped.

```
* feat(pubsub): implement GUAC ([#7428]), ([#7432]), ([#7436]), ([#7440]), ([#7449])

[#7428]: https://github.com/googleapis/google-cloud-cpp/pull/7428
[#7432]: https://github.com/googleapis/google-cloud-cpp/pull/7432
[#7436]: https://github.com/googleapis/google-cloud-cpp/pull/7436
[#7440]: https://github.com/googleapis/google-cloud-cpp/pull/7440
[#7449]: https://github.com/googleapis/google-cloud-cpp/pull/7449
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -97,20 +99,72 @@
...
+* fix(pubsub): do not overwrite ConnectionOptions ([#7406](https://github.com/googleapis/google-cloud-cpp/pull/7406))
```
Comment (dbolduc) [inline CHANGELOG.md:142]: This can be deleted, as it was included in the v1.32.1 change log.

**Review by coryan — COMMENTED** [review]
I like the changes that @dbolduc suggested.

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546#-:-MnRib3k6OJGRxS116dk:b-rgt024)* status: all files reviewed, 2 unresolved discussions (waiting on @dbolduc and @scotthart)

___
*[CHANGELOG.md, line 141 at r2](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546#-MnRV6JcvMPNH-gJz2qe-r2-141:-MnRiPqcCAVkom2lwuMu:b-8ayu6x) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/24cfa07172890a24aea6fb2427a04c25dab9820a/CHANGELOG.md#L141)):*
<details><summary><i>Previously, dbolduc (Darren Bolduc) wrote…</i></summary><blockquote>

These can probably be combined, and I think the common one can be dropped.

```
* feat(pubsub): implement GUAC ([#7428]), ([#7432]), ([#7436]), ([#7440]), ([#7449])

[#7428]: https://github.com/googleapis/google-cloud-cpp/pull/7428
[#7432]: https://github.com/googleapis/google-cloud-cpp/pull/7432
[#7436]: https://github.com/googleapis/google-cloud-cpp/pull/7436
[#7440]: https://github.com/googleapis/google-cloud-cpp/pull/7440
[#7449]: https://github.com/googleapis/google-cloud-cpp/pull/7449
```
</blockquote></details>

Agree

**Review by scotthart — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @coryan and @dbolduc)

___
*[CHANGELOG.md, line 141 at r2](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546#-MnRV6JcvMPNH-gJz2qe-r2-141:-MnRqd2jDj7m9ZK9DxI2:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/24cfa07172890a24aea6fb2427a04c25dab9820a/CHANGELOG.md#L141)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

Agree
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 142 at r2](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546#-MnRV6JcvMPNH-gJz2qe-r2-142:-MnRqdLh4kNoNBdJJEk5:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/24cfa07172890a24aea6fb2427a04c25dab9820a/CHANGELOG.md#L142)):*
<details><summary><i>Previously, dbolduc (Darren Bolduc) wrote…</i></summary><blockquote>

This can be deleted, as it was included in the v1.32.1 change log.
</blockquote></details>

Done.

**Review by coryan — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7546#-:-MnRs3uO45OhtChSDSns:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @dbolduc)

**Review by dbolduc — APPROVED** [review]

**Pull request merged**